Spikes/movement/Pointer.cpp

```diff
@@ -0,0 +1,91 @@
+#include "stdafx.h"
+#include "Pointer.h"
+
+//using namespace sf;
+
+
+
+
+//GLOBAL VARIABLES (FOR TESTING PURPOSES)
+const int PATH_RADIUS = 200;
+const float PI = std::acos(-1);
```

Comment:
this is not much of a calculation to do locally in methods, I understand your logic is creating game constants, but from now on just make them locally in methods that need them, I only count 2 methods every update. Later when a game constants header file is created we can redo it, for the purposes of spikes, no need